tests/test_java.py

```diff
@@ -42,8 +42,7 @@
     _application_docker_container,
     assert_function_in_collapsed,
     make_java_profiler,
-    snapshot_one_collapsed,
```

Comment:
Now that `snapshot_one_*` are not used - let's remove them?

Reply:
Removed in last commit